module/Application/src/Application/View/Helper/Acl.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Application\View\Helper;
+
+use Zend\View\Helper\AbstractHelper;
+
+
+class Acl extends AbstractHelper
+{
+
+    /**
+     * Acl
+     *
+     * @var \Zend\Permissions\Acl\Acl
+     */
+    protected $acl;
+
+    /**
+     * Get the current user's role.
+     *
+     * @return User|string
+     */
+    protected $role;
+
+    /**
+     * Check if a operation is allowed for the current role.
+     *
+     * @param string $operation Operation to be checked.
+     * @param string|ResourceInterface $resource Resource to be checked
+     *
+     * @return boolean
+     */
+    public function __invoke($operation, $resource)
```

Comment:
In the service, it is only that way because the resource may be left empty. And we don't actually use the resource often there at all, so I don't think that it will become confusing.